hw3/01-routing.js

```diff
@@ -14,38 +14,77 @@ const port = process.env.PORT || 5000;
 // For other routes, such as http://localhost:5000/other, this exercise should return a status code 404 with '404 - page not found' in html format
 
 const server = http.createServer((req, res) => {
-  const routes = [
-    'welcome',
-    'redirect',
-    'redirected',
-    'cache',
-    'cookie',
-    'check-cookies',
-    'other',
-  ];
-
-  let getRoutes = () => {
-    let result = '';
-
-    routes.forEach(
-      (elem) => (result += `<li><a href="/${elem}">${elem}</a></li>`)
-    );
-
-    return result;
-  };
-
-  if (req.url === '/') {
-    let routeResults = getRoutes();
-
-    res.writeHead(200, { 'Content-Type': 'text/html' });
-    res.write(`<h1>Exercise 01</h1>`);
-    res.write(`<ul> ${routeResults} </ul>`);
-    res.end();
-  }
-
-  // Add your code here
-});
+    const routes = [
+        'welcome',
+        'redirect',
+        'redirected',
+        'cache',
+        'cookie',
+        'check-cookies',
+        'other',
+    ];
+
+    const getRoutes = () => {
+        let result = '';
+
+        routes.forEach((elem) => (result += `<li><a href="/${elem}">${elem}</a></li>`));
 
+        return result;
+    };
+
+    if (req.url === '/') {
```

Comment:
This looks like a good opportunity for a switch statement

Reply:
I agree